src/core/interpreter/profile-parameter-validator.test.ts

```diff
@@ -497,16 +476,16 @@ describe('ProfileParameterValidator', () => {
         parameterValidator = new ProfileParameterValidator(ast);
       });
 
-      it('returns error for foo not being passed', () => {
-        expect(parameterValidator.validate({}, 'result', 'Test').isErr()).toBeTruthy();
+      it.each([undefined, null, 'value'])('returns ok for foo being %p', (value) => {
+        expect(parameterValidator.validate({ foo: value }, 'result', 'Test').isErr()).toBeTruthy();
       });
 
-      it('returns error for foo being undefined', () => {
-        expect(parameterValidator.validate({ foo: undefined }, 'result', 'Test').isErr()).toBeTruthy();
+      it('returns error for foo not being passed', () => {
```

Comment:
nitpick: here you use the word "passed" but in optional field you used the word "set"

Reply:
Fixed, thanks.